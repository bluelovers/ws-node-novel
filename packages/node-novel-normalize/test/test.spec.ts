import { normalize_strip, normalize_val } from '../index';

describe(`describe`, () =>
{

	let text = '00090_2章 不希望獨占的日子結束的面具工薪族版_二百四十四話　逐漸増加的春季預予于於定.txt';

	test(`normalize_strip`, () =>
	{

		let actual = normalize_strip(text);

		expect(actual).toMatchSnapshot();

	});

	test(`normalize_val`, () =>
	{

		let actual = normalize_val(text);

		expect(actual).toMatch(/244/);

	});

})
