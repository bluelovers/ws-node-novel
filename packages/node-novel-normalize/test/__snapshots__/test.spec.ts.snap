// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`describe normalize_strip 1`] = `"2章 不希望獨占的日子結束的面具工薪族版_二百四十四話　逐漸増加的春季預予于於定.txt"`;
